biggraphite/drivers/cassandra.py

```diff
@@ -1397,21 +1390,8 @@ def get_metric(self, metric_name, touch=False):
         if not uid or not config:
             return None
 
-        if touch:
-            # Small trick here: we also check that the parent directory
-            # exists because that's what we check to create the directory
-            # hierarchy.
-            # We do it only on 'touch==True' because we know that the caller
-            # accepts a small performance penalty already.
```

Comment:
I think this part was actually useful because it allows us to remove some entropy by recreating directories when needed.

Can you either keep it and always do it ?

Reply:
Done